src/modules/swap/pure/banners/TwapSuggestionBanner.tsx

```diff
@@ -0,0 +1,49 @@
+import { Currency, CurrencyAmount, Percent } from '@uniswap/sdk-core'
+
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components/macro'
+
+import { TradeUrlParams } from 'modules/trade/types/TradeRawState'
+import { parameterizeTradeRoute } from 'modules/trade/utils/parameterizeTradeRoute'
+
+import { Routes } from 'common/constants/routes'
+import { InlineBanner } from 'common/pure/InlineBanner'
+
+const StyledNavLink = styled(NavLink)`
+  color: inherit;
+  display: inline;
+  text-decoration: underline;
+
+  &:hover {
+    text-decoration: none;
+  }
+`
+
+export interface TwapSuggestionBannerProps {
+  priceImpact: Percent | undefined
+  buyingFiatAmount: CurrencyAmount<Currency> | null
+  tradeUrlParams: TradeUrlParams
+}
+
+const PRICE_IMPACT_LIMIT = 1 // 1%
+const AMOUNT_LIMIT = 50_000 // $50,000
```

Comment:
Use different amounts per network?

Reply:
@elena-zh already asked about it, waiting a response from Anxo